RELEASENOTES.md

```diff
@@ -30,7 +30,8 @@ By creating a [build authentication context](https://aka.ms/algosecrets#buildAut
       "settings": {
         "doNotPublishApps": false,
```

Comment:
```suggestion
        "doNotPublishApps": false,
        "useCompilerFolder": true,        
```
Include useCompilerFolder?

Reply:
This sample is different - the global settings are using compilerfolder and have set doNotPublishApps to true, because they might have dependencies, which cannot be resolved. When setting the buildAuthContext, you then set doNotPublishApps to false - meaning: Yes, publish the apps when using an online environment.

Reply:
Added a little extra text to the description